src/iceberg_functions/iceberg_scan.cpp

```diff
@@ -19,321 +19,659 @@
 
-	return std::move(list_extract_expr);
+    return std::move(list_extract_expr);
 }
 
 static unique_ptr<ParsedExpression> GetFilenameMatchExpr() {
-	auto data_colref_expr = make_uniq<ColumnRefExpression>("filename", "iceberg_scan_data");
-	auto delete_colref_expr = make_uniq<ColumnRefExpression>("file_path", "iceberg_scan_deletes");
+    auto data_colref_expr = make_uniq<ColumnRefExpression>("filename", "iceberg_scan_data");
+    auto delete_colref_expr = make_uniq<ColumnRefExpression>("file_path", "iceberg_scan_deletes");
 
-	auto data_filename_expr = GetFilenameExpr(std::move(data_colref_expr));
-	auto delete_filename_expr = GetFilenameExpr(std::move(delete_colref_expr));
+    auto data_filename_expr = GetFilenameExpr(std::move(data_colref_expr));
+    auto delete_filename_expr = GetFilenameExpr(std::move(delete_colref_expr));
 
-	return make_uniq<ComparisonExpression>(ExpressionType::COMPARE_NOT_DISTINCT_FROM, std::move(data_filename_expr),
-	                                       std::move(delete_filename_expr));
+    return make_uniq<ComparisonExpression>(ExpressionType::COMPARE_NOT_DISTINCT_FROM, std::move(data_filename_expr),
+                                           std::move(delete_filename_expr));
 };
 
 //! Uses recursive unnest on list of structs to return a table with all data and delete files
 //! TODO: refactor, probably.
 static unique_ptr<TableRef> MakeListFilesExpression(vector<Value> &data_file_values,
                                                     vector<Value> &delete_file_values) {
-	vector<Value> structs;
-	for (const auto &file : data_file_values) {
-		child_list_t<Value> child;
-		child.emplace_back(make_pair("file", file));
-		child.emplace_back(make_pair("type", Value("data")));
-		structs.push_back(Value::STRUCT(child));
-	}
-	for (const auto &file : delete_file_values) {
-		child_list_t<Value> child;
-		child.emplace_back(make_pair("file", file));
-		child.emplace_back(make_pair("type", Value("delete")));
-		structs.push_back(Value::STRUCT(child));
-	}
-
-	// Unnest
-	vector<unique_ptr<ParsedExpression>> unnest_children;
-	unnest_children.push_back(make_uniq<ConstantExpression>(Value::LIST(structs)));
-	auto recursive_named_param = make_uniq<ConstantExpression>(Value::BOOLEAN(true));
-	recursive_named_param->alias = "recursive";
-	unnest_children.push_back(std::move(recursive_named_param));
-
-	// Select node
-	auto select_node = make_uniq<SelectNode>();
-	vector<unique_ptr<ParsedExpression>> select_exprs;
-	select_exprs.emplace_back(make_uniq<FunctionExpression>("unnest", std::move(unnest_children)));
-	select_node->select_list = std::move(select_exprs);
-	select_node->from_table = make_uniq<EmptyTableRef>();
-
-	// Select statement
-	auto select_statement = make_uniq<SelectStatement>();
-	select_statement->node = std::move(select_node);
-	return make_uniq<SubqueryRef>(std::move(select_statement), "iceberg_scan");
+    vector<Value> structs;
+    for (const auto &file : data_file_values) {
```

Comment:
The majority of lines changed here are still formatting changes - could you [format using our clang-format config](https://github.com/duckdb/duckdb/blob/main/.clang-format)?

Reply:
@Mytherin fixed the formatting, sorry about that!